data.json

```diff
@@ -39,6 +39,10 @@
     "SupportsVendorExperiment": [
       "go1.5", "go1.5.1", "go1.5.2", "go1.5.3", "go1.5.4",
       "go1.6", "go1.6.1", "go1.6.2", "go1.6.3", "go1.6.4"
+    ],
+    "SupportsModuleExperiment": [
+      "go1.11beta2", "go1.11beta3", "go1.11rc1", "go1.11rc2",
+      "go1.11"
```

Comment:
It would be nice to make it scalable for go1.11+, e.g. symver parse the version etc.

Reply:
I have thought the same thing, but both semver and arrays like these are both solutions to the same problem. Since it will only be an experiment for several versions I have been hesitant to integrate a semver bash library into the buildpack. It may be worth considering though. If you would like to PR an implementation like that I'd love to review it to see if/how it makes this and similar structures easier to maintain in the buildpack.